src/main/java/org/candlepin/subscriptions/tally/TallySnapshotController.java

```diff
@@ -22,11 +22,9 @@
 
 import io.micrometer.core.annotation.Timed;
 import java.time.OffsetDateTime;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please don't use wildcard imports for classes (static method wildcard imports are okay).  Let me know if you need help configuring your editor to avoid these.

Reply:
Thanks for the suggestion I have changed that setting.